lms/static/scripts/frontend_apps/components/SubmitGradeForm.js

```diff
@@ -215,10 +172,10 @@ export default function SubmitGradeForm({ student }) {
               ref={inputRef}
               onInput={handleKeyDown}
               type="input"
-              defaultValue={grade}
+              defaultValue={grade.data ?? ''}
```

Comment:
This sent me down a bit of a rabbit hole because I realized that, while intuitive in its naming, I didn't know where the heck the `defaultValue` prop came from here (I knew it wasn't an HTML attribute on `input` elements). It's just a [React construct](https://reactjs.org/docs/uncontrolled-components.html#default-values) to account for component lifecycle (as I'm sure you are already aware). I presume it does something undesirable if if set to `null` or `undefined`?

Reply:
I think `undefined` would work the same here. `null` isn't allowed by `defaultValue`'s type. This is a string input field is logically we want the fallback, if no grade has been assigned, to be an empty string though.